compose_api.go

```diff
@@ -92,6 +96,45 @@ func (f StackIdentifier) String() string {
 	return string(f)
 }
 
+type ComposeStackReaders []io.Reader
+
+func (r ComposeStackReaders) applyToComposeStack(o *composeStackOptions) {
+	currentDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	// choose directory to keep temporary files
+	// like
+	// 		/tmp/testcontainers-go/my-awesome-service-2f686f6d652f676f666f7262726f6b65
+	projectName := filepath.Base(currentDir)
+	projectHash := fmt.Sprintf("%x", fnv.New32a().Sum([]byte(currentDir)))[:32]
+	tmpDir := filepath.Join(os.TempDir(), "testcontainers-go", fmt.Sprintf("%s-%s", projectName, projectHash))
+	if err := os.RemoveAll(tmpDir); err != nil {
+		panic(err)
+	}
```

Comment:
Is this needed to avoid collisions? I think we can skip this, or at least log the error but continuing

Reply:
Yes, tried to keep directory clean and don't mixed it with another files in project-name-collision cases.
I removed this part.